packages/commonjs/test/fixtures/function/dynamic-require-resolve-index/_config.js

```diff
@@ -1,4 +1,4 @@
-const nodeResolve = require('@rollup/plugin-node-resolve');
+const { nodeResolve } = require('@rollup/plugin-node-resolve');
```

Comment:
Why have you switched to importing the named export in favor of the default export?
Is it just to be explicit or didn't it work anymore with the default export?
I know that it exports both, but we need to be sure that nothing has been fatally broken :-)

Reply:
This is not an `import`, this is a CJS `require`. Since a recent version, this plugin switched to named export mode. We could also have used the default export at `.default` but then if we need to use a property anyway, I prefer the named export.